packages/demo/.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 node_modules
 public/build/
+yarn.lock
```

Comment:
We need the yarn.lock file to be checked in to the repo, so please remove this from the gitignore.